docs/webapp/webapp_exp_table.md

```diff
@@ -1,78 +1,81 @@
+     * Task name
+     * Git - Optional fields for checking out the code from a git repository: 
        * Repository URL
        * Version specification - one of the following:
          * Tag 
          * Branch 
          * Commit ID
-       * Execution Entry Point
+       * Entry Point - The code to run
          * Working Directory
-         * One of the following
-           * Script name
-           * Module (see [python module specification](https://docs.python.org/3/using/cmdline.html#cmdoption-m)) 
-       * Add `Task.init` call - If selected, [`Task.init()`](../references/sdk/task.md#taskinit) call is added to the 
-       entry point. Select if it is not already called within your code
+         * Script type - Python/Shell 
+         * Binary - The binary executing the script (e.g. python3, bash etc).
+         * Type – How the code is provided
+           * Script - The name of the file to run using the above specified binary
+           * Module - The name of a python module to run (Python only, see [Python module specification](https://docs.python.org/3/using/cmdline.html#cmdoption-m))
+           * Custom code - Directly provide the code to run. Write code, or upload a file:
+             * File name - The script in which your code is stored. Click `Upload` to upload an existing file.
+             * Content - The actual code. Click `Edit` to modify the script’s contents.
+       * Add `Task.init` call (Python only) - If selected, a [`Task.init()`](../references/sdk/task.md#taskinit) call is automatically added to 
+       your script (Use if if you script does not yet make use of ClearML)
    * **Arguments** (*optional*) - Add [hyperparameter](../fundamentals/hyperparameters.md) values.
-   * **Environment** (*optional*) - Set up the experiment’s python execution environment using either of the following 
-   options:
-     * Use Poetry specification - Requires specifying a docker image for the experiment to be executed in.
-     * Manually specify the python environment configuration:
-       * Python binary - The python executable to use
-       * Preinstalled venv - A specific existing virtual environment to use. Requires specifying a docker image for the 
-       experiment to be executed in.
-       * Python package specification:
-         * Skip - Assume system packages are available. Requires specifying a docker image for the experiment to be 
-         executed in.
-         * Use an existing `requirements.txt` file
-         * Explicitly specify the required packages
-   * **Docker** (*optional*) - Specify Docker container configuration for executing the experiment
-     * Image - Docker image to use for running the experiment
-     * Arguments - Add Docker arguments as a single string
-     * Startup Script - Add a bash script to be executed inside the Docker before setting up the experiment's environment 
+   * **Environment** (optional) - Set up the task’s execution environment
```

Comment:
@pollfly How come 'optional' is italicized for Arguments  but not for Environment?